refinery/templates/base.html

```diff
@@ -322,6 +322,9 @@
   <script src="{% static "vendor/jquery-file-upload/js/jquery.fileupload-angular.js" %}"></script>
   <script src="{% static "js/data-set-import.js" %}"></script>
 
+  <!-- Fetch external SVGs referenced in <use> elements when the browser fails to do so -->
+  <script defer src="{% static "vendor/svgxuse/svgxuse.js" %}"></script>
```

Comment:
Yeah locally it might not be an issue because everything loads fast :) The question is wether svgxuse needs to be loaded before `<use>` are parsed.

Also, is "vendor/svgxuse/svgxuse.js" the minified version?